From 7f01e6365be00301b1edec7b79c9d72bf07d9eaa Mon Sep 17 00:00:00 2001
From: Janne Grunau <j@jannau.net>
Date: Sun, 24 Aug 2025 21:32:20 +0200
Subject: [PATCH 1/1] configs: apple: decrease boot delay to 1 second

On Apple silicon machines u-boot's primary use is providing an UEFI boot
environment. It's not expected that AUTOBOOT is interrupted but it's
still desired to provide a clue that it is possible. Thios allows
booting from USB storage devices.

Signed-off-by: Janne Grunau <j@jannau.net>
---
 configs/apple_m1_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/apple_m1_defconfig b/configs/apple_m1_defconfig
index 057313938e1..28b96aff662 100644
--- a/configs/apple_m1_defconfig
+++ b/configs/apple_m1_defconfig
@@ -4,6 +4,7 @@ CONFIG_ARCH_APPLE=y
 CONFIG_DEFAULT_DEVICE_TREE="apple/t8103-j274"
 CONFIG_SYS_BOOTM_LEN=0x800000
 CONFIG_SYS_LOAD_ADDR=0x0
+CONFIG_BOOTDELAY=1
 # CONFIG_AUTOBOOT_KEYED is not set
 # CONFIG_AUTOBOOT_MENU_SHOW is not set
 CONFIG_BOOTCOMMAND="bootflow scan -b"
